### DISCUSSION TITLE
InputNumber 设置范围的最大值与最小值

### DISCUSSION THREAD

**Discussion by jingshou** [nutui-uniapp/nutui-uniapp#411] [General]
uniapp+nutui 的NutInputNumber 设置最大值与最小值，失去焦点后不会自动校验。
![image](https://github.com/user-attachments/assets/2c227789-cbae-4710-8f6d-28c8072b488d)
与官网的版本：4.3.12 不一致的。
![image](https://github.com/user-attachments/assets/e6c24015-4896-4029-a2d8-1ce6d24aa649)

**Comment by xiaohe0601** [comment]
组件缺陷可以提issue，discussions没有消息提示可能会漏掉
